packages/errors/src/index.ts

```diff
@@ -25,6 +26,17 @@ export const createErrorType = (
     }
   );
 
+export class ThrottlingException extends Error {
+  readonly code: string;
+
+  constructor() {
+    super('ThrottlingException');
+    this.name = 'ThrottlingException';
+    this.code = 'ThrottlingException';
```

Comment:
So this whole class is only necessary to set the `code` on the error object?

Reply:
Yes.